wordpress-mobile/WordPress-iOS
# Update secrets

Done via `bundle exec fastlane run cofigure_update`.

Internal ref paaHJt-1Qj.

To test:

Run `bundle exec fastlane run configure_apply` and verify your local copy of the files in this diff changed. There may be other files changed, too, depending on how long it's been since you've run the command.


PR submission checklist:
